php/ext/google/protobuf/message.c

```diff
@@ -1024,7 +1035,15 @@ static void hex_to_binary(const char* hex, char** binary, int* binary_len) {
 PHP_METHOD(Any, __construct) {
   init_file_any(TSRMLS_C);
   MessageHeader* intern = UNBOX(MessageHeader, getThis());
-  custom_data_init(any_type, intern PHP_PROTO_TSRMLS_CC);
+
```

Comment:
@TeBoring All of these constructors are the same. Could they use a common function instead of repeating the same code?

Reply:
Updated in my branch. Please sync again.